rotkehlchen/data_migrations/migrations/migration_8.py

```diff
@@ -65,5 +67,11 @@ def data_migration_8(rotki: 'Rotkehlchen', progress_handler: 'MigrationProgressH
                 message_type=WSMessageType.EVM_ADDRESS_MIGRATION,
                 data=[{'evm_chain': str(x[0]), 'address': x[1]} for x in to_add_accounts],
             )
+    else:
+        # The only step is update the spam assets
```

Comment:
nitpick to not waste space

```suggestion
    else:  # The only step is updating the spam assets
```